docs/openapi.json

```diff
@@ -148,13 +148,13 @@
   },
   "components" : {
     "schemas" : {
-      "Version" : {
-        "required" : [ "value" ],
+      "FileHandle" : {
+        "required" : [ "handle" ],
         "type" : "object",
         "properties" : {
-          "value" : {
+          "handle" : {
             "type" : "string",
-            "description" : "version identifier of the current API definition"
+            "description" : "Reference to an uploaded file. This can be e.g. a PDF or a picture (e.g. JPG) of a contract or a purchase receipt."
```

Comment:
Rephrase to "Reference to an uploaded document. This can be a PDF or a picture (e.g. JPG) of a contract or a purchase receipt."